scripts/hid_configurator/README.rst

```diff
@@ -42,6 +42,9 @@ Complete the following instructions, depending on your operating system:
 * `Windows`_
 * `Debian/Ubuntu/Linux Mint`_
 
+You also need to ensure that the `Linux Vendor Firmware Service (LVFS) <LVFS_>`_ ``fwupd`` deamon is not running in the background.
```

Comment:
I think `daemon` not deamon + other occurrences

Reply:
Just fixed it :)